ultra/ultra/train.py

```diff
@@ -231,8 +287,8 @@ def train(
                     break
 
         while not dones["__all__"]:
-            # Break if any of the agent's step counts is 1000000 or greater.
-            if any([episode.get_itr(agent_id) >= 1000000 for agent_id in agents]):
+            # Break if any of the agent's step counts is 10000000 or greater.
+            if any([episode.get_itr(agent_id) >= 10000000 for agent_id in agents]):
```

Comment:
I was noticing that experiments were pre-maturely ending due to that restriction in some grade-based training runs. I believe instead of hardcoding the max-steps, we should convert into a flag

Reply:
I will create another PR if we implement the flag modification, because it will affect the curriculum branch and the ultra develop branch as well